app/uk/gov/hmrc/cdsreimbursementclaimfrontend/journeys/RejectedGoodsSingleJourney.scala

```diff
@@ -128,6 +128,20 @@ final class RejectedGoodsSingleJourney private (
       .map(_.collect { case (taxCode, Some(amount)) => (taxCode, amount) })
       .getOrElse(Map.empty)
 
+  def getNextNdrcDetailsToClaim: Option[NdrcDetails] =
+    answers.reimbursementClaims
+      .map(_.collect { case (taxCode, None) =>
+        taxCode
+      })
+      .toList
+      .flatten
+      .map(getNdrcDetailsFor)
+      .flatten(Option.option2Iterable)
+      .headOption
+
+  def allReimbursementAmountEntered: Boolean =
```

Comment:
we already have `hasCompleteReimbursementClaims` method for that check

Reply:
I have removed the duplicate method.